src/main/java/edu/byu/cs240/checkstyle/readability/CommentedCode.java

```diff
@@ -1,24 +1,25 @@
 package edu.byu.cs240.checkstyle.readability;
 
-import com.puppycrawl.tools.checkstyle.api.AbstractFileSetCheck;
-import com.puppycrawl.tools.checkstyle.api.FileText;
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
 
-import java.io.File;
-import java.util.Arrays;
-import java.util.List;
 import java.util.Set;
 
 /**
  * Reports successive lines of commented out code
  *
  * @author Michael Davenport
  */
-public class CommentedCode extends AbstractFileSetCheck {
+public class CommentedCode extends AbstractCheck {
 
     private static final Set<Character> CODE_LINE_END_CHARS = Set.of(';', ',', '{', '}', '(', ')', '/');
```

Comment:
The set of chars feels a little heavy on the syntax in the event of adding a new symbol.

I took a stab at converting it into a simpler format, but I don't think it turned out any better (Java doesn't have concise ways of inserting data into different kinds of collections. This is my attempt, for the record, but I think we should just leave it as is and press forward:

```java

    /** While within a comment block, commented code is detected based on a line ending in any of these symbols */
    private static final String CODE_LINE_END_CHARS_STR = ";,{}()/";
    private static final Set<Character> CODE_LINE_END_CHARS =
                CODE_LINE_END_CHARS_STR.chars()
                .mapToObj(c -> (char) c)
                .collect(Collectors.toSet());
```

I like having the dense view of all the chars, but I don't love the process required to convert them into the set at eval time.